crates/utils/src/task_manager/tasks.rs

```diff
@@ -6,6 +6,11 @@
 //                    \        /      |      \       /
 //                   WATCH_TASK  CONF_CHANGE  LOG_PERSIST
 
+// In integration tests, we use the bottom tasks, like `WatchTask`, `ConfChange`, and `LogPersist`,
```

Comment:
Add a "NOTE:" prefix to highlight this comment.

Reply:
Done